src/main/webapp/index.html

```diff
@@ -0,0 +1,13 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Is this needed for index.html? Also doesn't seem like the right commenting style in .html files.